packages/template-retail-react-app/app/pages/account/wishlist/index.test.js

```diff
@@ -349,61 +19,53 @@ jest.mock('commerce-sdk-react-preview', () => {
     }
 })
 
-beforeEach(() => {
-    jest.resetModules()
+jest.mock('../../../hooks/use-current-customer', () => {
+    return {
+        useCurrentCustomer: jest.fn().mockReturnValue({
+            data: {
+                customerId: 'some_id',
+                isRegistered: true
+            }
+        })
+    }
 })
```

Comment:
Once we merge https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1052 to the feature branch, we might not need this mock.